plugins/filter_log_to_metrics/log_to_metrics.c

```diff
@@ -729,6 +759,43 @@ static int cb_log_to_metrics_init(struct flb_filter_instance *f_ins,
     }
     ctx->input_ins = input_ins;
 
+
+    if (ctx->interval_sec <= 0) {
+        ctx->interval_sec = atoi(DEFAULT_INTERVAL_SEC);
```

Comment:
Could you please use `strtol` instead and change these properties to be `long` to match?

Reply:
Of course I can change that. I just copied that from the other plugins, because it was used like this everywhere in all the other plugins with atoi 
(e.g: https://github.com/fluent/fluent-bit/blob/4ac5b7b4214b79c1f550c3e634e2dcf46d26f212/plugins/in_exec/in_exec.c#L280 or https://github.com/fluent/fluent-bit/blob/4ac5b7b4214b79c1f550c3e634e2dcf46d26f212/plugins/in_calyptia_fleet/in_calyptia_fleet.c#L2184 or https://github.com/fluent/fluent-bit/blob/4ac5b7b4214b79c1f550c3e634e2dcf46d26f212/plugins/in_random/random.c#L121)